src/main/lrsql/admin/interceptors/lrs_management.clj

```diff
@@ -0,0 +1,26 @@
+(ns lrsql.admin.interceptors.lrs-management
+  (:require [clojure.spec.alpha :as s]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [io.pedestal.interceptor.chain :as chain]
+            [lrsql.admin.protocol :as adp]))
+
+(def validate-delete-actor-params
+  (interceptor {:name ::validate-delete-actor-params
+                :enter (fn validate-delete-params [ctx]
+                         (let [params (get-in ctx [:request :json-params])]
+                           (if-some [err (s/explain-data (s/keys :req-un [:lrsql.spec.actor/actor-ifi]) params)]
+                             (assoc (chain/terminate ctx)
+                                      :response
+                                      {:status 400
+                                       :body {:error (format "Invalid parameters:\n%s"
+                                                             (-> err s/explain-out with-out-str))}})
+                             (assoc ctx :actor-ifi params))))}))
+
+(def delete-actor
+  (interceptor
+   {:name ::delete-actor
+    :enter (fn delete-actor [ctx]
+             (let [{lrs :com.yetanalytics/lrs
+                    actor-ifi :actor-ifi}
+                   ctx]
+               (adp/-delete-actor lrs actor-ifi)))}))
```

Comment:
This interceptor is not returning the context/making a response

Reply:
fixed